core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -486,7 +528,7 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
         origin: string;
         data: {
             action: CUSTOMER_ACTIONS;
-            payload: ActionPayload | SetUrlPayload | Row[] | ContentletArea;
+            payload: ActionPayload | SetUrlPayload | Row[] | HeadlessContentletArea;
```

Comment:
As mentioned @fmontes  about the `headless` word, check if here applies too.

Reply:
It does apply, I'm thinking in a better name